c10/core/thread_pool.cpp

```diff
@@ -6,7 +6,7 @@ namespace c10 {
 ThreadPool::ThreadPool(
     int pool_size,
     int numa_node_id,
-    std::function<void()> init_thread)
+    const std::function<void()>& init_thread)
```

Comment:
Err, how do this aligns with https://clang.llvm.org/extra/clang-tidy/checks/modernize/pass-by-value.html

Reply:
Reverted

Reply:
We have no chance to move the function because it is passed to lambda in a loop